<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPdfUnify</class>
 <widget class="QMainWindow" name="QPdfUnify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPdfUnify</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUnify">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Unify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFile1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>40</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFile1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>File 1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFile1">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>20</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOutput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>40</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leOutput">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>140</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFile2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>40</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFile2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>File 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFile2">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>60</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>368</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
